### PR TITLE
fix breadcrumb styling

### PR DESCRIPTION
**Pull request by songgao**
Before and after:

screen shot 2018-08-15 at 11 06 22 am

### CODE REVIEW

**Comment by songgao** [comment]
Please hold off reviewing. Fixing ...

**Comment by songgao** [comment]
OK ready again!

File: shared/fs/header/breadcrumb.desktop.js [hunk 1]
```diff
@@ -11,91 +18,118 @@ type Props = {
...
+    isElectron: {
+      ...Styles.globalStyles.flexBoxRow,
+      left: 0,
+      right: 0,
+      flexGrow: 1,
+      alignItems: 'center',
+      paddingLeft: 16,
+      paddingRight: 16,
+      height: 24,
+    },
```
Comment (jzila) [inline shared/fs/header/breadcrumb.desktop.js]: which of these require `isElectron`?

**Comment by nathunsmitty** [comment]
awww yiss
download

Although could this maybe be an iconfont thing? cc @thebearjew

**Comment by songgao** [comment]
@nathunsmitty Thanks for the funny comic 😂I don't think iconfont was broken. This is likely from a previous refactor.

**Comment by songgao** [comment]
@nathunsmitty actually I might have jumped the gun. It seems the download cards are broken too. Hmm.

**Pull request merged**